### PR TITLE
Slack bot innovation week

### PR DESCRIPTION
**Pull request by Blasz**
This PR makes a few changes to support sending slack notifications when the landkid status of your PR changes.
The main changes are:

* Emitting a new `'LAND_REQUEST.STATUS.CHANGED'` event when a land request status is updated. We can then add an event listener in our own instance for it that then sends a request to a new `af-notifications` service (will create a PR for this later) that is responsible for sending the slack notification.
* Sending & storing atlassian account IDs of both PR authors and land request triggerers. The current `aaid` fields that we send & store were actually changed to bitbucket UUIDs a very long time ago but were never renamed to reflect this.

Some things to call out:

* The real aaid fields are labelled `accountId` to differentiate them from the existing misnamed aaid fields. Renaming the fields would be a migration headache.
* For the same migration headache reasons, the new fields have been configured to allow null values. This means we don't have to update existing entries in our databases with non-null values and follows the pattern of previous migrations we've added.
* We need to re-install the landkid bitbucket connect add-on as the service descriptor has been updated to send the actual aaid of the triggerer in the land requests.

### CODE REVIEW

File: src/db/index.ts [hunk 1]
```diff
@@ -108,6 +124,20 @@ export class LandRequest extends Model<LandRequest> implements ILandRequest {
         { transaction: t },
       );
     });
+    eventEmitter.emit('LAND_REQUEST.STATUS.CHANGED', {
+      landRequestId: this.id,
+      pullRequestId: this.pullRequestId,
+      buildId: this.buildId,
+      author: this.pullRequest?.authorAccountId,
```
Comment (RajaBellebon) [inline src/db/index.ts:131]: is there a difference between account_id and authorAccountId

File: src/lib/Runner.ts [hunk 2]
```diff
@@ -187,11 +187,11 @@ export class Runner {
         dependencies.map((queueItem) => queueItem.request.pullRequestId).join(', ');
     }
 
-    await landRequest.setStatus('running', depPrsStr);
-
     // Todo: these should really be functions on landRequest
     landRequest.buildId = buildId;
     landRequest.dependsOn = dependsOnStr;
+    await landRequest.setStatus('running', depPrsStr);
```
Comment (RajaBellebon) [inline src/lib/Runner.ts:193]: any reason for moving the function?

File: src/lib/Runner.ts [hunk 3]
```diff
@@ -490,11 +491,17 @@ export class Runner {
...
-    });
+    return LandRequest.create<LandRequest>(
+      {
+        triggererAaid: landRequestOptions.triggererAaid,
+        triggererAccountId: landRequestOptions.triggererAccountId,
+        pullRequestId: pr.prId,
+        forCommit: landRequestOptions.commit,
+      },
+      {
+        include: [PullRequest],
```
Comment (RajaBellebon) [inline src/lib/Runner.ts:502]: are we adding more metadata?

File: src/routes/api/index.ts [hunk 4]
```diff
@@ -261,9 +261,13 @@ export function apiRoutes(runner: Runner, client: BitbucketClient, config: Confi
         prId,
         // Must always have a valid aaid, so lets set it to Luke's aaid
         triggererAaid: req.user ? req.user.aaid : '557057:9512f4e4-3319-4d30-a78d-7d5f8ed243ae',
+        triggererAccountId: req.user
+          ? req.user.accountId
+          : '557057:9512f4e4-3319-4d30-a78d-7d5f8ed243ae',
         commit: prInfo.commit,
         prTitle: prInfo.title,
         prAuthorAaid: prInfo.authorAaid,
+        prAuthorAccountId: prInfo.author,
```
Comment (RajaBellebon) [inline src/routes/api/index.ts:270]: any reason to not be `authorAccountId`

**Review by RajaBellebon — APPROVED** [review]
The changes lgtm - left few minor questions

**Review by luciaquirke — APPROVED** [review]

**Pull request merged**